Lib/fontTools/misc/encodingTools.py

```diff
@@ -18,6 +18,7 @@
 		# See
 		# https://github.com/fonttools/fonttools/issues/236
 		0: { # Macintosh, platEncID==0, keyed by langID
```

Comment:
Since it is write that, I am wondering if we should add all the langID from: https://docs.microsoft.com/en-us/typography/opentype/spec/name#macintosh-language-ids

The problem is that I don't know the encoding for each of them